types.tsx

```diff
@@ -209,17 +208,22 @@ export type Ingredient = {
   amount: number;
   unit: string;
   meta: never[] | string[];
-  metaInformation: never[] | string[]
+  metaInformation: never[] | string[];
   measures: {
     us: {
       amount: number;
       unitShort: string;
       unitLong: string;
-    },
+    };
     metric: {
-      amount: number
+      amount: number;
       unitShort: string;
       unitLong: string;
+<<<<<<< HEAD
```

Comment:
I think you missed another merge conflict?

Just need to accept the RecipeScreenInfo and fix the curly braces.